runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowPipelineTranslator.java

```diff
@@ -318,6 +318,18 @@ public Job translate(List<DataflowPackage> packages) {
 
       WorkerPool workerPool = new WorkerPool();
 
+      // If streaming engine is enabled set the proper experiments so that it is enabled on the
+      // back end as well.
+      if (options.isEnableStreamingEngine()) {
```

Comment:
Hey, while you are here, please change line 357 to 
if (options.isStreaming() && isStreamingEngineEnabled(options)) { ...

Reply:
See above